react/week3/todo-app/src/components/GetTodos.js

```diff
@@ -0,0 +1,61 @@
+import { useState, useEffect } from "react";
+import TodoList from "./TodoList";
+import AddTodo from "./AddTodo"
+
+const GetTodos = () => {
```

Comment:
I'm not sure the `GetTodos`-component and the `TodoList` component need to be two separate components.
I think you can move the `todoArray.map(...)` in here, if you want.
If you choose not to go this way, I'd rename this component to something along the lines of `TodosHandler` or `TodosContainer`.

Reply:
> I'm not sure the `GetTodos`-component and the `TodoList` component need to be two separate components.
> I think you can move the `todoArray.map(...)` in here, if you want.
> If you choose not to go this way, I'd rename this component to something along the lines of `TodosHandler` or `TodosContainer`.

i want to rename this component...tq